scripting-docs/javascript/javascript-in-vs-2017.md

```diff
@@ -200,6 +200,34 @@ You could use the following `tsconfig.json` to make sure the language service on
   }
 }
 ```
+# Troubleshooting The JavaScript language service has been disabled for the following project(s)
+When opening a JavaScript project that has a huge amount of content you might get a message that reads "The JavaScript language service has been disabled for the following project(s)". The most common reason for having so much .js source is due to including libraries with a ton of content.
+
+A simple way to optimize your project, is by adding a `tsconfig.json` file on your project root to let the language service know which files are safe to ignore. Use the sample below to exclude the most common directories where libraries are stored:
+
+```json
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true,
+    "maxNodeModuleJsDepth": 2,
+    "noEmit": true,
+    "skipLibCheck": true
+  },
+  "exclude": [
+    "**/bin",
+    "**/bower_components",
+    "**/jspm_packages",
+    "**/node_modules",
+    "**/obj",
+    "**/platforms"
+  ]
+}
+```
+
+Add more directories as you see fit. Some other examples include "vendor" or "wwwroot/lib" directories.
+
+More details on [Language Service Disabled](https://billti.github.io/jsdocs/articles/troubleshooting/index.html#language-service-is-disabled).
```

Comment:
If this is the official VS docs, then please don't add a link to an external private (and temporary) repo being used for drafts.

Reply:
Removed paragraph.